CodeStarterCamp_Week1/main.swift

```diff
@@ -8,33 +8,44 @@
 
 import Foundation
 
-// Step 2 (modified)
-
-func makeIcecreamBody(bodywidth: Int, bodyheight: Int) {
-    for _ in 1...bodyheight {
-        for _ in 1...bodywidth {
-            print("*", terminator: "")
-        }
-        print()
-    }
+// Step 3 (1)
+func makeBar(barlength: Int) {
+    for _ in 1...barlength {
+        print(" | |")}}
+
+func makeBodyRow(bodyshape: String, topping: String) -> String {
+    return topping + bodyshape + topping
 }
 
-func makeIcecreamBar(bodywidth: Int, barheight: Int) {
-    for _ in 1...barheight {
-        if bodywidth % 2 != 0 {
-            for _ in 1...((bodywidth + 1) / 2) - 2 {
-                print(" ", terminator: "")
-            }
-            print("| |")
-        }
-        if bodywidth % 2 == 0 {
-            for _ in 1...(bodywidth / 2) - 1 {
-                print(" ", terminator: "")
-            }
-            print("||")
-        }
-    }
+var bodyRow: (String, String) -> String = makeBodyRow(bodyshape:topping:)
+
+func makeBodyCol(bodylength: Int, bodyrow: Any) {
+    for _ in  1...bodylength {
+            print(bodyrow)
+}}
```

Comment:
in 다음에 띄어쓰기는 한번만! 그리고 print문의 뎁스가 잘못되었네요. 더불어, 중괄호 역시 개행을 해주시는게 좋습니다.
```suggestion
    for _ in 1...bodylength {
        print(bodyrow)
    }
}
```
제가 컨밴션을 이렇게 자세하게 말씀드리는 이유는, 코드의 컨벤션이 통일되어있어야 다른 사람이 코드를 읽을 때 헷갈리지 않기 때문입니다. 사람마다 코드 컨벤션은 다를 수 있으니 제가 제시드린대로 꼭 바꾸실 필요까진 없습니다.
다만, 일단은 기본적인 컨벤션을 먼저 익히는게 순서라고 생각합니다. 먼저 통일된 코드 컨벤션을 지키는 습관을 먼저 가지고 다영님만의 컨벤션을 만들어보세요!